nio/block/mixins/persistence/persistence.py

```diff
@@ -39,14 +39,14 @@ def persisted_values(self):
     def _load(self):
         """ Load the values from persistence
 
-        Item is loaded from persistence using block name, once item
+        Item is loaded from persistence using block id, once item
         is loaded, all persisted values are examined against loaded
         item, if it exists, such value is then set as an attribute
         in the block instance
         """
         self.logger.debug("Loading from persistence")
         # load whole item from persistence
-        item = self._persistence.load(self.name(), default={})
+        item = self._persistence.load(self.label(include_id=True), default={})
```

Comment:
We should probably just persist off the `id` only, right? It's rarely read by humans and by using the label we would lose the persistence if the block was renamed.

Reply:
good call!